CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+#### Changes
+
+#### Fixes
+* Java: Fix XRange/XRevRange with a negative count ([#1920](https://github.com/valkey-io/valkey-glide/pull/1920))
```

Comment:
nit: 
```suggestion
* Java: XRange/XRevRange should return null instead of `GlideException` when given a negative count ([#1920](https://github.com/valkey-io/valkey-glide/pull/1920))
```

Reply:
This should be the PR name then